.changeset/smooth-zoos-share.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
lol - you may be able to ignore `.github/` as well?

Reply:
It _should_ be... 🤔  

<img width="745" alt="image" src="https://github.com/Khan/perseus/assets/77138/844c592d-62f9-439e-bfee-74ddb8a47204">

Reply:
Maybe it needs to be an exclude_glob instead to make it work? Whereas maybe these are just checking exact matches?

Reply:
Just trying to figure out why you had to add this changeset for this PR, since you shouldn't have had to if this was configured correctly!

Reply:
I wasn't 100% sure. I was thinking that maybe GH runs the action using the `.github` files from the base branch and not the PR?

Reply:
And with regards to the various `excludes*` inputs, any file that matches any of the exclusions is skipped.

Reply:
@jeresig I figured it out!  [Fixed](https://github.com/Khan/actions/pull/49) in `khan/actions` and tested [here](https://github.com/Khan/perseus/pull/531).